modules/ROOT/partials/configuration/tinycomments_fetch_author_info.adoc

```diff
@@ -0,0 +1,61 @@
+[[tinycomments_fetch_author_info]]
+== `+tinycomments_fetch_author_info+`
+
+The {pluginname} plugin uses the `+tinycomments_fetch_author_info+` callback function to provide author details for comments displayed in the editor.
+
+This function is called whenever a comment is rendered, and it allows the editor to display the author’s name and avatar based on the current user.
+
+The `+tinycomments_fetch_author_info+` function is passed a request (`+req+`) object as the first parameter, and a `+done+` callback as the second.
+
+The function must call `+done+` with an object containing the following properties:
+
+* `+author+` (`+string+`): A unique identifier for the user.
+* `+authorName+` (`+string+`): The full name to display for the user.
+* `+authorAvatar+` (`+string+`): A URL pointing to the user's avatar image.
+
+.Example of `+tinycomments_fetch_author_info+` function
+[source,js]
+----
+const tinycomments_fetch_author_info = (_, done) => {
+  setTimeout(() => done({
+	  author: currentUser.id, // Maps to the `author` field in the comment object
+	  authorName: currentUser.fullName, // Maps to the `authorName` field in the comment object
+	  authorAvatar: currentUser.image, // Maps to the `authorAvatar` field in the comment object
+  }), fakeDelay);
+};
+----
+
+.Example of `+tinycomments_fetch_author_info+` function with a custom author
+[source,js]
+----
+const currentUser = {
+  id: 'johnsmith',
+  name: 'John Smith',
+  fullName: 'John Smith',
+  description: 'Company Founder',
+  image: '{{imagesdir}}/logos/android-chrome-192x192.png'
+};
+
+const tinycomments_fetch_author_info = (_, done) => {
+  setTimeout(() => done({
+    author: currentUser.id, // Maps to the `author` field in the comment object
+    authorName: currentUser.fullName, // Maps to the `authorName` field in the comment object
+    authorAvatar: currentUser.image, // Maps to the `authorAvatar` field in the comment object
+  }), fakeDelay);
+};
```

Comment:
I think it would be better to have a call to some fake api instead of a `setTimeout`. Something like:
```suggestion
const tinycomments_fetch_author_info = (_, done) => {
  fetch('https://example.api.com/currentUser')
    .then(res => res.json())
    .then(currentUser => {
      done({
        author: currentUser.id, // Maps to the `author` field in the comment object
        authorName: currentUser.fullName, // Maps to the `authorName` field in the comment object
        authorAvatar: currentUser.image, // Maps to the `authorAvatar` field in the comment object
      })
    })
};
```

Reply:
Updating with this with the inclusion of missing semi-colons 😄 
```suggestion
const tinycomments_fetch_author_info = (_, done) => {
  fetch('https://example.api.com/currentUser')
    .then(res => res.json())
    .then(currentUser => {
      done({
        author: currentUser.id, // Maps to the `author` field in the comment object
        authorName: currentUser.fullName, // Maps to the `authorName` field in the comment object
        authorAvatar: currentUser.image, // Maps to the `authorAvatar` field in the comment object
      });
    });
};
```